hack/image-stream-build-config.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: "Template"
+metadata:
+  name: cluster-logging-dev-build-template
+  annotations:
+    description: "Template for creating local builds of logging components from source."
+    tags: "infrastructure"
+labels:
+  logging-infra: development
+  provider: openshift
+  component: development
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: cluster-logging-operator
+    name: cluster-logging-operator
+  spec: {}
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      app: cluster-logging-operator
+    name: cluster-logging-operator
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: cluster-logging-operator:latest
+    resources: {}
+    source:
+      git:
+        uri: ${CL_OP_GITHUB_URL}
+        ref: ${CL_OP_GITHUB_BRANCH}
+      type: Git
+    strategy:
+      dockerStrategy:
+        dockerfilePath: Dockerfile
+      type: Docker
+parameters:
+-
+  description: 'URL for cluster-logging-operator fork'
+  name: CL_OP_GITHUB_URL
```

Comment:
Any reason not to simplify this `GITHUB_URL` and `GITHUB_BRANCH` or

Reply:
I wanted them to be consistent with the environment variable names - we have to use a unique prefix `CL_OP` or `ES_OP` for the environment variables because we will typically set both sets of env. vars. at the same time.